BookingsAPI/Bookings.AcceptanceTests/Steps/SuitabilityAnswersSteps.cs

```diff
@@ -0,0 +1,94 @@
+        private readonly SuitabilityAnswerEndpoints _suitabilityEndpoints = new ApiUriFactory().SuitabilityAnswerEndpoints;
+        private readonly HearingsEndpoints _hearingEndpoints = new ApiUriFactory().HearingsEndpoints;
+
+        public SuitabilityAnswersSteps(TestContext acTestContext)
+        {
+            _acTestContext = acTestContext;
+        }
+
+        [Given(@"I have the suitable answers submitted")]
+        public void GivenIHaveTheSuitableAnswersForParticipants()
+        {
+            AddSuitabilityAnswersToParticipants(_acTestContext.HearingId, _acTestContext.Participants);
+        }
+
+        [Given(@"I have a request to get the suitable answers")]
+        public void GivenIHaveARequestToGetTheSuitableAnswers()
+        {
+            _acTestContext.Request = _acTestContext.Get(_suitabilityEndpoints.GetSuitabilityAnswers(""));
+        }
+
+        [Given(@"I have a request to the second set of suitable answers")]
+        public void GivenIHaveARequestToTheSecondSetOfSuitableAnswers()
+        {
+            //Create hearing and submit answers to participants
+            AddSuitabilityAnswersToHearing();
+            AddSuitabilityAnswersToHearing();
+            
+            //Get first set of the suitability answers
+            var suitabilityGetRequest = _acTestContext.Get(_suitabilityEndpoints.GetSuitabilityAnswerWithLimit("", 1));
+            var response = _acTestContext.Client().Execute(suitabilityGetRequest);
+            var firstSetOfRecords = ApiRequestHelper.DeserialiseSnakeCaseJsonToResponse<SuitabilityAnswersResponse>(response.Content);
+
+            //Get 2nd set of the suitability answers
+            _acTestContext.Request = _acTestContext.Get(_suitabilityEndpoints.GetSuitabilityAnswerWithLimit(firstSetOfRecords.NextCursor, 1));
+        }
+
+
+        [Then(@"suitable answers should be retrieved")]
+        public void ThenSuitableAnswersShouldBeRetrieved()
+        {
+            var model = ApiRequestHelper.DeserialiseSnakeCaseJsonToResponse<SuitabilityAnswersResponse>(_acTestContext.Json);
+            model.Should().NotBeNull();
+            model.PrevPageUrl.Should().NotBe(model.NextPageUrl);
+            model.ParticipantSuitabilityAnswerResponse.Should().NotBeNull();
+            model.ParticipantSuitabilityAnswerResponse.Count.Should().BeGreaterThan(0);
+            var participantAnswer = model.ParticipantSuitabilityAnswerResponse[0];
+            participantAnswer.FirstName.Should().NotBeEmpty();
+            participantAnswer.LastName.Should().NotBeEmpty();
+            participantAnswer.HearingRole.Should().NotBeEmpty();
+            participantAnswer.Answers.Should().NotBeNull();
+            participantAnswer.Answers.Count.Should().BeGreaterThan(0);
+            participantAnswer.UpdatedAt.Should().Be(model.ParticipantSuitabilityAnswerResponse.Max(s => s.UpdatedAt));
+        }
+
+        private void AddSuitabilityAnswersToHearing()
```

Comment:
maybe name the method to match more what it does?

Reply:
You mean, instead of AddSuitabilityAnswersToHearing, rename it to AddSuitabilityAnswersToParticipantsOfAHearing

Reply:
Yes, something like that. It actually creates a hearing as well right?

Reply:
Well yes, we need a hearing for most of the scenarios